hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/Router.java

```diff
@@ -308,6 +308,22 @@ protected void serviceInit(Configuration configuration) throws Exception {
     }
   }
 
+  private void checkRouterId() {
```

Comment:
Add a javadoc saying that we set the router if not set to prevent XYZ.

Reply:
Thanks for your advice.